ansible/roles/common/tasks/main.yml

```diff
@@ -49,6 +54,18 @@
   when: "'metrics' in groups"
   notify:
     - restart collectd
+- name: install OMS Agent
```

Comment:
Would it be possible to push this to an Azure specific file?

Reply:
Acknowledged. We are working on a commit update which will make all of these monitoring agent related changes, Azure-specific. We'll likely do a (conditional) include file if there are many lines.

Reply:
That would be nice because I think it will help with the maintainability of Muchos.

Reply:
@arvindshmicrosoft should an issue be opened for this for follow on work or were you thinking of doing that in this PR?  I am trying to resolve conversations.

Reply:
For now, we have added conditions to filter for cluster_type == 'azure' and also opened #272 to track similar refactoring tasks for the future.